realm/realm-library/src/objectServer/java/io/realm/RealmUser.java

```diff
@@ -432,8 +442,9 @@ public RealmMongoDBService getMongoDBService() {
         return null;
     }
 
+    @SuppressFBWarnings("NP_METHOD_PARAMETER_TIGHTENS_ANNOTATION")
```

Comment:
What does this annotation do?

Reply:
It disables a FindBugs false positive. You can use this if what FindBugs complains about isn't accurate.